practices/managing-static-assets.md

```diff
@@ -0,0 +1,20 @@
+# Managing static assets
+
+By **static assets** we mean webfonts, JavaScript libraries (like jQuery or Mathjax), and any other files that get loaded by the browser from a domain that is not ours. These static assets are characterised by:
+
+* Being essential or important for the working of a page.
+* Being updated infrequently.
+* Require some manual intervention in order to be updated.
```

Comment:
Not sure of this characterisation, as some will be made by build tools... what is the intention here?

Reply:
The reason for creating the document is to address the problem that we've seen recently with some resources being blocked from China, so this document is specific to the third party resources that are let's say "first class citizens" of our sites and are required in order to get part of the functionality. Things like webfonts, jQuery, MathJax, and any other libraries we may have.

By "built tools" I think that you mean concatenated JS and CSS files? If that's the case, I wasn't thinking about these while writing this document, I'm assuming that we always serve those from our own servers.

Reply:
I was thinking more of sprite sheets and other UI images. I guess it's the extent of the "manual intervention" that's confusing me...

Reply:
I see it's not very clear in my explanation. By "manual intervention" I'm thinking of instances where the app links to `${CDN}/jquery-3.2.0.min.js` (and not to `${CDN}/jquery-latest.min.js`), so in order to upgrade a dev has to replace the link destination with `${CDN}/jquery-3.2.1.min.js` or whatever.

So, not a result of a build process, not a result of an automated tool.

Reply:
Aha ok, well the example you cite of webfonts are I think an example of where manual intervention is not required. So we could say "Possibly require some manual intervention..." or just drop the point, as I'm not sure how relevant it is to the point we are trying to make? Or am I missing something?